README.md

```diff
@@ -62,17 +62,20 @@ Advanced composition
 Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you have a `MyFormInput` component that should also act as both a `Row` and a `Col`.
 
 ```js
-import {getRowClassNames, getColClassNames} from 'react-flexbox-grid'
+import {getRowProps, getColumnProps} from 'react-flexbox-grid'
 
 export default function MyFormInput(props) {
+  const { className: rowClassName } = getRowProps(props);
+  const { className: colClassName, ...myProps } = getColumnProps(props);
```

Comment:
I would rather avoid destructuring in this example, I think many people using this library are beginners. Some people will know about destructuring, but then renaming (`className: rowClassName`) is another hurdle to clear.

I would write it like this:

```js
const rowProps = getRowProps(props);
const colProps = getColumnProps(props);

return (
  <form className={rowProps.className}>
    {/* ... */}
  </form>
);
```